src/server/utils/onModuleLoad.js

```diff
@@ -91,6 +87,45 @@ export default function onModuleLoad({
       pwa,
       errorPageUrl,
       dnsCache,
+      redirectAllowList,
+    } = {},
+    [META_DATA_KEY]: metaData,
+  } = module;
+  validateCspIsPresent(csp);
+  clearModulesUsingExternals();
+  if (provideStateConfig) {
+    setStateConfig(provideStateConfig);
+  }
+  if (errorPageUrl) {
+    setErrorPage(errorPageUrl);
+  }
+  if (redirectAllowList) {
+    setRedirectAllowList(redirectAllowList);
```

Comment:
what happens when a root module has a redirect allow list, then a new version gets released without a redirect allow list? this needs to ensure that its reset